### DISCUSSION TITLE
Будет ли исправлен режим мир?

### DISCUSSION THREAD

**Discussion by gat3v** [Sokol-Games-GmbH/PixelCombats.PublicApi#147] [Q&A]
**Имеется ввиду получение блоки с лопатой.**

_в оффициальном Мирном режиме присутствует ошибка или же не продуманность, когда игрок использует баг с лопатой и заходя на сервер и за синих он получает комнатный инвентарь или же inventory, не убирается потому что убирает у синих инвентарь но его кидает вне команды и даёт инвентарь комнатный._

_Для решение надо на inventory.Melee.Value и inventory.Build.Value вместо true поставить false._

**Comment by gat3v** [comment]
**До исправления Создан режим под ID 1734, там исправлено и функционально, но видимо ничего не помогло чтобы игроки создавали его.**

**Comment by kkohno** [comment]
можете сделать в мирном режиме ветку и сделать пул реквест?
вот ссылка на репу мирного режима если что.

https://github.com/kkohno/PixelCombats.GameModes.Peace
